aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -47,6 +47,19 @@ module aptos_token::token {
     const EWITHDRAW_ZERO: u64 = 22;
     const ENOT_TRACKING_SUPPLY: u64 = 23;
     const ENFT_NOT_SPLITABLE: u64 = 24;
+    const ECOLLECTION_NAME_TOO_LONG: u64 = 25;
```

Comment:
These need error messages

Reply:
sure, I can add one for mine?  idk what the others mean